src/accordion/accordion-item.tsx

```diff
@@ -37,13 +38,14 @@ export const AccordionItem = ({
     // =============================================================================
 
     useEffect(() => {
-        setExpand(expanded);
-    }, [expanded]);
```

Comment:
If you remove this, we cannot expand it mid way already. This should still be there.

Reply:
Instead since the purpose of this is to expand it on initial load if the `expandAll` is otherwise, then we should do this
```
 useEffect(() => {
        if (hasFirstLoad) setExpand(expanded);
    }, [expanded]);
```